app/index.html

```diff
@@ -2,6 +2,7 @@
 <html ng-app="app" ng-strict-di="true" manifest="/coriolis.appcache">
 <head>
   <title ng-bind="title">Coriolis</title>
+  <meta charset="UTF-8">
```

Comment:
This is not needed because the web server sets the content header in the response.

Google recommends that this tag is actually omitted due to performance reasons

http://diywpblog.com/avoid-a-character-set-in-the-meta-tag/